src/controllers/challenges/index.ts

```diff
@@ -122,7 +122,7 @@ export const challengeRouter = async function (app: FastifyInstance) {
 					{ $inc: { billy_bucks: -amount, "metrics.gambling.challenges.bets": 1 } }
 				)
 			]);
-			return result;
+			return { bet, billyBucks: user.billy_bucks };
```

Comment:
this is fine but im pretty sure most of our other endpoint return snake_case

Reply:
ah yes, good catch. thanks